contributing/standards/markdown.md

```diff
@@ -105,7 +105,9 @@ Exercism is a very intentional product - things are there because they've been d
 
 ## Comments
 
-- Prfer Markdown comments instead of HTML comments (e.g. use `[comment]: # (Actual comment...)` rather than `<!-- Actual comment -->`
+- Prefer Markdown comments instead of HTML comments (e.g. use `[comment]: # (Actual comment...)` rather than `<!-- Actual comment -->`
```

Comment:
Will HTML comments be supported anyway? Markdown comments are not equivalent, you can't add them inline for example.

Reply:
It'll be contextual. There are places where I do not want to trust the markdown we're parsing, so `<!-- -->` would be sanitized to `&lt;-- --&gt;` before being sent to the browser. For others I'm fine with HTML in general, and there are some that are in the middle.

Also, a lot of our markdown files are intended to be consumed **as markdown** (e.g. the exercise intros), so adding **any** comments into those should be considered bad practice, as they're not hidden from students.

For example, analyzer comments (where this originated), I don't massively see the need to have comments in them, other than for other maintainers to see when looking at the website-copy repo, in which case they can sit at the top of the files I think.

Reply:
> Also, a lot of our markdown files are intended to be consumed as markdown (e.g. the exercise intros), so adding any comments into those should be considered bad practice, as they're not hidden from students.

I may be misremembering things, but wasn't there previous discussion if the CLI could strip them?

Reply:
Yeah - it probably could actually. Good point.

Reply:
I find it really useful to add comments meant for the maintainers in documents like the exercise intro to add context for _why_ I did something the way I did, so I'd prefer it if they were allowed in there.